### ISSUE TITLE
/repeat command

### ISSUE THREAD

**Issue by Tanmay-901** [Tanmay-901/Crypto_Tracker_Bot#1]
complete the "repeat" command feature, removing the possible bugs and adding a "stop" command to undo this.

**Comment by blank2121** [comment]
hey so you want a command that will share the current value of a crypto currency after a set interval? and you want the stop command to just stop the repeat command correct? also do you need this to be done soon.

**Comment by Tanmay-901** [comment]
Hi @blank2121, yes the bot will keep on sending the pricelist of crypto-currencies added in the list after the /repeat command at the desired time interval and the /stop command is only to stop the /repeat command.
As for your second ques, there's no such timeline as of now.

**Comment by Winston-cs** [comment]
Ok I'll take a look today. do you want me to fork the project and make a new branch?

**Comment by Winston-cs** [comment]
another thing about the dependencies you are using. I do not know some of ones you are using, so can you tell me if they run asynchronously  by default or do i have to add that in so the rest of the code can run?

**Comment by Tanmay-901** [comment]
> Ok I'll take a look today. do you want me to fork the project and make a new branch?

Yes please, anyhow this will be a new release with a lot of restructuring after this feature, so make a branch(recomended) or a directory whatever suits you.

**Comment by Tanmay-901** [comment]
> another thing about the dependencies you are using. I do not know some of ones you are using, so can you tell me if they run asynchronously by default or do i have to add that in so the rest of the code can run?

the asynchronous property hasn't been tested yet since the bot supports only one common coinlist for all end users as of now. This is another enhancement in the backlog waiting to be addressed since I haven't been able to put in time to this project for a long time now.

**Comment by Winston-cs** [comment]
are you ok if i implement a solution via threading? like, i would have a loop running in a different thread that can be terminated at any time with the stop command but it would send the updated value of the crypto currencies at each interval. also do you want me to fork and make a new branch?

**Comment by Tanmay-901** [comment]
Sure @blank2121, if you think that'll work without compromising the response time then please go ahead.
yes, please fork and make a new branch with a suitable name.

**Comment by Winston-cs** [comment]
@Tanmay-901 because there are no comments on what the code does, can you please tell me what part of the code needs to be ran at the desired interval. Also what are the inputs to the variables? Like it is just a simple int of the amount of minutes?
